### DISCUSSION TITLE
doris be 崩溃，1.2.5版本

### DISCUSSION THREAD

**Discussion by kiddingl** [apache/doris#27011] [RFC]
`*** Query id: 21a9926b0986474d-a68cd2640616545b ***
*** Aborted at 1699985035 (unix time) try "date -d @1699985035" if you are using GNU date ***
*** Current BE git commitID: Unknown ***
*** SIGSEGV address not mapped to object (@0x0) received by PID 449 (TID 0x7f4dd21ca700) from PID 0; stack trace: ***
 0# doris::signal::(anonymous namespace)::FailureSignalHandler(int, siginfo_t*, void*) at /root/doris/be/src/common/signal_handler.h:420
 1# os::Linux::chained_handler(int, siginfo*, void*) in /usr/local/sbin/java-1.8.0-openjdk-1.8.0.332.b09-1.el7_9.x86_64/jre/lib/amd64/server/libjvm.so
 2# JVM_handle_linux_signal in /usr/local/sbin/java-1.8.0-openjdk-1.8.0.332.b09-1.el7_9.x86_64/jre/lib/amd64/server/libjvm.so
 3# signalHandler(int, siginfo*, void*) in /usr/local/sbin/java-1.8.0-openjdk-1.8.0.332.b09-1.el7_9.x86_64/jre/lib/amd64/server/libjvm.so
 4# 0x00007F4ED9BAC400 in /lib64/libc.so.6
 5# je_arena_dalloc_promoted at ../src/arena.c:1604
 6# __pthread_create_2_1 in /lib64/libpthread.so.0
 7# std::thread::_M_start_thread(std::unique_ptr<std::thread::_State, std::default_delete<std::thread::_State> >, void (*)()) at ../../../../../libstdc++-v3/src/c++11/thread.cc:150
 8# doris::DataConsumerPool::get_consumer_grp(doris::StreamLoadContext*, std::shared_ptr<doris::DataConsumerGroup>*) at /root/doris/be/src/runtime/routine_load/data_consumer_pool.cpp:73
 9# doris::RoutineLoadTaskExecutor::exec_task(doris::StreamLoadContext*, doris::DataConsumerPool*, std::function<void (doris::StreamLoadContext*)>) at /root/doris/be/src/runtime/routine_load/routine_load_task_executor.cpp:267
10# std::_Function_handler<void (), std::_Bind_result<void, void (doris::RoutineLoadTaskExecutor::*(doris::RoutineLoadTaskExecutor*, doris::StreamLoadContext*, doris::DataConsumerPool*, doris::RoutineLoadTaskExecutor::submit_task(doris::TRoutineLoadTask const&)::{lambda(doris::StreamLoadContext*)#1}))(doris::StreamLoadContext*, doris::DataConsumerPool*, std::function<void (doris::StreamLoadContext*)>)> >::_M_invoke(std::_Any_data const&) at /var/local/ldb-toolchain/include/c++/11/bits/std_function.h:291
11# doris::PriorityThreadPool::work_thread(int) at /root/doris/be/src/util/priority_thread_pool.hpp:146
12# execute_native_thread_routine at ../../../../../libstdc++-v3/src/c++11/thread.cc:84
13# start_thread in /lib64/libpthread.so.0
14# clone in /lib64/libc.so.6

/opt/workspace/be/bin/start_be.sh: line 315:   449 Segmentation fault      ${LIMIT:+${LIMIT}} "${DORIS_HOME}/lib/doris_be" "$@" 2>&1 < /dev/null`
![image](https://github.com/apache/doris/assets/36329336/3c1ddfd2-f66e-4170-8552-df4d0b4f06ea)

怎么解决呀，哪里出了问题了呢

**Comment by LemonLiTree** [comment]
try to upgrade
